docs/outline.toml

```diff
@@ -449,6 +449,11 @@ title = "Notifications and Callback Tools"
 module = "prefect.utilities.notifications"
 functions = ["callback_factory", "slack_notifier", "gmail_notifier"]
 
+[pages.utilities.jira_notification]
```

Comment:
As jira is not imported as standard, it needs to be imported as an extra.  We thought moving it to its own page would allow us to raise a warning if jira is not already installed for the jira notifier without affecting the other notifiers.

Reply:
Gotcha; so whether it requires an extra or not actually doesn't matter for the auto-generated documentation -- I'm able to patch any missing imports so that I can still extract the relevant docstrings.  If you put it in the same page as the others, you'd need to add a 
```python
try:
    from .jira import jira_notifier # might not be 100%
except ImportError:
    pass
```
into `utilities.notifications.__init__.py`

Up to you though, if you want it on its own documentation page that's OK